docs/build

```diff
@@ -7,6 +7,14 @@ cd "${BASH_SOURCE%/*}"
 test -t 1 && OPTS='-it' || OPTS=''
 
 SPHINX_IMAGE=${SPHINX_IMAGE:-ghcr.io/trinodb/build/sphinx:3}
+CONTAINER=docker
 
-docker run --rm $OPTS -e TRINO_VERSION -u $(id -u):$(id -g) -v "$PWD":/docs $SPHINX_IMAGE \
+if ! [ -x "$(command -v docker)" ]; then
+    if [ -x "$(command -v podman)" ]; then
```

Comment:
Instead of nesting this and hence defaulting to podman as choice and failing when it is not installed either ..

lets add podman to an elif statement

and in the else show an error message about no suitable container runtime found or so.

Reply:
Thank you for the review @mosabua . I have updated the PR based on your recommendations.